sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTestTest.java

```diff
@@ -162,6 +162,349 @@ void testDown() throws TimeoutException {
         storage.close();
     }
 
+    /**
+     * Tests whether calling the test without any input data causes a failure.
+     */
+    @Test
+    public void testRequirementR00550BadNoTestData() {
+        assertThrows(NoTestData.class, testClass::testRequirementR00550);
+    }
+
+    /**
+     * Test whether calling the test when all ReportParts of all DescriptionModificationReports have
+     * ModificationTypes != Crt causes a failure.
+     */
+    @Test
+    public void testRequirementR00550BadNoDescriptionModificationReportsWithReportPartsWithModificationTypeCrt()
+            throws Exception {
```

Comment:
exception is missing in javadoc

Reply:
Added it to the Javadoc.

Reply:
Blank line between description and tag missing.

Reply:
Blank line added.